pkgs/applications/misc/avalonia-ilspy/default.nix

```diff
@@ -84,8 +86,7 @@ buildDotnetModule rec {
       ln -s $out/bin/ILSpy $out/Applications/ILSpy.app/Contents/MacOS/ILSpy
     '';
 
-  dotnet-sdk = dotnetCorePackages.sdk_6_0;
-  dotnet-runtime = dotnetCorePackages.runtime_6_0;
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
```

Comment:
It's not needed, `buildDotnetModule` already uses the runtime from the SDK by default so it's redundant.

Reply:
@GGG-KILLER When is it it needed? I thought it's a good practice as an example package from the docs sets `dotnet-runtime` and I got similar reviews from you and maybe other people. Sorry for the noise, just want to understand dotnet with nix better :)

Reply:
I'd say it's necessary when using `combinePackages`, asp.net core or a newer runtime than the SDK provides.

If you like to be explicit about it you can specify the runtime manually, but it's not really necessary anymore, it used to be because it defaulted to `pkgs.dotnet-runtime`, but now it defaults to `dotnet-sdk.runtime`.